requirements.txt

```diff
@@ -1,5 +1,5 @@
 django>=1.8,<1.9
-djangorestframework>=3.1,<3.2
+git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
```

Comment:
Why are we using this drf commit? If there is any necessity then is there any plan to upgrade to stable release than one particular commit ?